app/src/i18n/locales/en/onboarding.json

```diff
@@ -2,30 +2,49 @@
   "onboarding-heading": "Let's Start Your Emissions Inventory!",
   "onboarding-details": "We need to set and confirm the city where you want to do the inventory.<br/>You are one step closer to getting your GPC emissions inventory!",
   "start-button": "Start City Inventory Profile",
-
-  "setup-heading": "Select City and Year",
+  "city": "City",
+  "setup-city-heading": "Select City",
+  "setup-city-details": "Search and select the city or municipality for which you want to create your GHG emission inventory.",
   "go-back": "Go Back",
+  "continue": "Continue",
   "setup-step": "Setting up your Inventory",
+  "setup-inventory-details-heading": "Set up inventory details",
+  "setup-inventory-details-description": "Provide some basic details about the inventory you want to create. You can edit these settings afterwards.",
+  "inventory-goal": "Inventory goal",
+  "inventory-goal-description": "Want to learn more about these inventory formats? <2>Learn more</2> about the GPC Framework.",
+  "gwp-heading": "Global Warming Potential (GWP)",
+  "gwp-description": "If this is your first inventory, we recommend using AR6 (latest version) for your inventory calculations. If your city has previous inventories, use the same version as before. <2>Learn more</2> about GWP.",
```

Comment:
It would be good to notify the product team or Evan of the new translation strings for onboarding.